src/test/java/seedu/address/logic/commands/AddCommandTest.java

```diff
@@ -47,6 +50,20 @@ public void execute_duplicatePerson_throwsCommandException() {
         assertThrows(CommandException.class, AddCommand.MESSAGE_DUPLICATE_CONTACT, () -> addCommand.execute(modelStub));
     }
 
+    @Test
+    public void execute_validWeddingIndices_throwsException() throws Exception {
```

Comment:
test name should be execute_invalidWeddingIndices_throwsException?

Reply:
done!